bindings/python/crocoddyl/multibody/impulse-costs/com-position.hpp

```diff
@@ -0,0 +1,58 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2018-2019, LAAS-CNRS
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef BINDINGS_PYTHON_CROCODDYL_MULTIBODY_IMPULSE_COSTS_COM_POSITION_HPP_
+#define BINDINGS_PYTHON_CROCODDYL_MULTIBODY_IMPULSE_COSTS_COM_POSITION_HPP_
+
+#include "crocoddyl/multibody/impulse-costs/com-position.hpp"
+
+namespace crocoddyl {
+namespace python {
+
+namespace bp = boost::python;
+
+void exposeImpulseCostCoM() {
+  bp::class_<ImpulseCostModelCoM, bp::bases<ImpulseCostModelAbstract> >(
+      "ImpulseCostModelCoM", bp::init<boost::shared_ptr<StateMultibody>, boost::shared_ptr<ActivationModelAbstract> >(
+                                 bp::args("self", "state", "activation"),
+                                 "Initialize the CoM position cost model.\n\n"
+                                 "For this case the default nu is equals to model.nv.\n"
+                                 ":param state: state of the multibody system\n"
+                                 ":param activation: activation model"))
+      .def(bp::init<boost::shared_ptr<StateMultibody> >(
+          bp::args("self", "state"),
+          "Initialize the CoM position cost model.\n\n"
+          "For this case the default activation model is quadratic, i.e.\n"
+          "crocoddyl.ActivationModelQuad(3), and nu is equals to model.nv.\n"
+          ":param state: state of the multibody system"))
+      .def("calc", &ImpulseCostModelCoM::calc_wrap, bp::args("self", "data", "x"),
+           "Compute the CoM position cost.\n\n"
+           ":param data: cost data\n"
+           ":param x: time-discrete state vector")
+      .def<void (ImpulseCostModelCoM::*)(const boost::shared_ptr<ImpulseCostDataAbstract>&, const Eigen::VectorXd&,
```

Comment:
You will need to adapt the bindings. For instance, the code will depend on CostDataAbstract